hnn_core/dipole.py

```diff
@@ -27,7 +27,13 @@ def simulate_dipole(net, n_trials=None, record_vsoma=False,
     record_isoma : bool
         Option to record somatic currents from cells
     postproc : bool
-        If False, no postprocessing applied to the dipole
+        If True, smoothing (``dipole_smooth_win``) and scaling
+        (``dipole_scalefctr``) values are read from the parameter file, and
+        applied to the dipole objects before returning. Note that this setting
+        only affects the dipole waveforms, and not somatic voltages, possible
+        extracellular recordings etc. The preferred way is to use the
+        :meth:`~hnn_core.dipole.Dipole.smooth` and
+        :meth:`~hnn_core.dipole.Dipole.scale` methods instead. Default: False.
```

Comment:
should we raise a deprecation warning? good to get started with good habits ...

Reply:
Sure, what are you thinking though? Something like

```python
warnings.warn('The postproc-argument will be removed in a '
              'future version of hnn-core. Please define '
              'smoothing and scaling explicitly.',
               DeprecationWarning)
```

Reply:
Are we actually planning to get rid of `postproc`? I think the warning would only apply for people who don't pass it as a named argument.

Reply:
take a look here:

https://github.com/mne-tools/mne-python/blob/ed64dc7bd0750bab2e99a1cfcbce2928eb37e202/doc/install/contributing.rst#deprecate-with-a-decorator-or-a-warning

I think we could do the same way.

> Are we actually planning to get rid of postproc

I think so? We do need to give a `whats_new` entry saying all data returned will now be unprocessed unlike in the past when it was all processed/smoothed

Reply:
I agree about getting rid of it! Will you add a commentary in what's_new @jasmainak ?

Reply:
oh sorry I just saw this. See the example I shared from MNE :)